ui/tests/acceptance/settings/mount-secret-backend-test.js

```diff
@@ -298,4 +300,61 @@ module('Acceptance | settings/mount-secret-backend', function (hooks) {
       `${v1} navigates to list route`
     );
   });
+
+  module('Community', function (hooks) {
+    hooks.beforeEach(function () {
+      this.version = this.owner.lookup('service:version');
+      this.version.type = 'community';
+      this.store = this.owner.lookup('service:store');
+    });
+
+    test('identity_token_key form field does not show even when an oidc key exists', async function (assert) {
```

Comment:
Can you explain how having an oidc key affects the logic? From what I can tell, it's just checking that the _attr_  `identityTokenKey` exists in the form logic, not that it's checking for an actual key 🤔

Reply:
Good call out. Updating test.